cfme/tests/cli/test_appliance_console.py

```diff
@@ -1291,7 +1290,14 @@ def test_appliance_console_extend_storage_negative():
             2.
             3. Check there is no extra disk present.
     """
-    pass
+    command_set = ("ap", RETURN, "11")
+    result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    assert [i for i in result[-1].split("\n") if "1) /dev/sr0: 0 MB" in i.strip()] != [], (
```

Comment:
Can you just do?
```suggestion
    assert [i for i in result[-1].split("\n") if "1) /dev/sr0: 0 MB" in i.strip()]
```
A small comment about this assertion may also be helpful.